cluster/gce/config-default.sh

```diff
@@ -145,6 +145,16 @@ ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 # TODO(piosz) remove this option once Metrics Server became a stable thing.
 ENABLE_METRICS_SERVER="${KUBE_ENABLE_METRICS_SERVER:-true}"
 
+# Optional: Metadata agent to setup as part of the cluster bring up:
+#   none        - No metadata agent
+#   stackdriver - Stackdriver metadata agent
+# Metadata agent is a daemon set that provides metadata of kubernetes objects
+# running on the same node for exporting metrics and logs.
+ENABLE_METADATA_AGENT="${KUBE_ENABLE_METADATA_AGENT:-none}"
+
+# Version tag of metadata agent
+METADATA_AGENT_VERSION="${KUBE_METADATA_AGENT_VERSION:-0.2-0.0.13-5}"
```

Comment:
Caution, this image does not incorporate the performance updates using the watch implementation.

Reply:
The correct image tag to use here is 0.2-0.0.13-5-watch if we're going to go with the latest version of the metadata agent that implements watch.

Reply:
The purpose of env variable is to overwrite this value without the need of change in OSS code. I can update the tag in a follow up PR, to keep it more up-to-date.